woocommerce/woocommerce-gateway-stripe
# feat: account `wp/data` scaffolding

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1873

This will be needed for onboarding, as well as the main settings page.

- Adds a controller to retrieve data from the Stripe account
- Adds the `account` redux slice (used to determine the account capabilities)
- Adds the `accountKeys` redux slice (used for onboarding)

# Testing instructions
This is mainly scaffolding. However, you can still test some of its functionality:
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Ensure you have a connected account
- Go to the settings page: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- Open the browser console, type `wp.data.select('wc/stripe').getAccountKeys()`
- In the network tab you should see a request to a `wp-json/wc/v3/wc_stripe/account_keys` endpoint
- Open the browser console, type `wp.data.select('wc/stripe').getAccount()`
- In the network tab you should see a request to a `wp-json/wc/v3/wc_stripe/account` endpoint
